bin/init_new_gem

```diff
@@ -72,6 +72,71 @@ put base / 'manifest.yaml', <<~'YAML'
   #   - name: pathname
 YAML
 
+put base.parent / '.rubocop.yml', <<~'YAML'
+  # This configuration inherits from /.rubocop.yml.
+  # You can configure RBS style of this gem.
+  # This file is used on CI. It is configured to automatically
+  # make rubocop suggestions on pull requests for this gem.
+  # If you do not like the style enforcement, you should remove this file.
+  inherit_from: ../../.rubocop.yml
+
+  ##
+  # If you want to customize the style, please consult with the gem reviewers.
+
+  RBS/Layout/CommentIndentation:
```

Comment:
I think it is not a good idea to list all cops in this file, because we need to update this script manually when the gem is updated.
How about writing a link to the documentation of the cops list instead of listing all cops here?

Reply:
I was concerned about the cost of increasing the version of rubocop-on-rbs.

In rubocop, new cop is supposed to warn with `Enabled: pending`. We decided to solve the problem by doing this operation in rubocop-on-rbs as well.
https://docs.rubocop.org/rubocop/versioning.html#pending-cops